### DISCUSSION TITLE
Does neovim have a plugin like emac's projectile?

### DISCUSSION THREAD

**Discussion by kartikdc** [NvChad/NvChad#1715] [Q&A] [answered]
This is the projectile plugin I am talking about: https://github.com/bbatsov/projectile

I don't know the full scope of what it does, but it keeps track of projects (git repositories) that I worked on. After that I could open it using some functions that the plugin provided (kinda like telescope's find_files but for projects).

Is there anything like this for neovim? The reason I am looking for something like this is I want to be able to open neovim once and switch between projects easily.

I don't using NvChad but thought I would ask here anyways. Sorry for the noise.

**Comment by siduck** [comment, marked as answer]
i only know about this plugin https://github.com/dunstontc/projectile.nvim

**Reply by kartikdc** [reply]
Also found this: https://github.com/ahmedkhalf/project.nvim

**Comment by dgmora** [comment]
For switching between files there's also https://github.com/tpope/vim-projectionist
